server/src/controllers/Events/resolver.ts

```diff
@@ -324,6 +324,19 @@ export class EventResolver {
       where: { user_id_event_id: { user_id: userId, event_id: eventId } },
       include: { event: true },
     });
+    const userInfos = await prisma.users.findUnique({
+      where: { id: userId },
+    });
+
+    const eventInfos = await prisma.events.findUnique({
+      where: { id: eventUser.event_id },
+    });
+
+    await new MailerService({
+      emailList: [userInfos?.email],
```

Comment:
And wanted data can be taken from `eventUser`.
```suggestion
      emailList: [eventUser.user.email],
```